RHEL/7/input/checks/templates/template_kernel_module_disabled

```diff
@@ -0,0 +1,86 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_kernel_modules_disabled.py.  DO NOT EDIT.  -->
+  <definition class="compliance"
+  id="kernel_module_KERNMODULE_disabled" version="1">
+    <metadata>
+      <title>Disable KERNMODULE Kernel Module</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>The kernel module KERNMODULE should be disabled.</description>
+      <reference source="swells" ref_id="20130928" ref_url="test_attestation"/>
```

Comment:
Use your name / credentials here, and update date stamp, the template was created. While the template is derived from the existing RHEL-6 one, it's completely new template (as indicated by ```version=1``` field). Proper code attribution is important (don't be afraid to use your name there, since it was your contribution).